groups.go

```diff
@@ -8,8 +8,7 @@ import (
 
 // Group contains all the information for a group
 type Group struct {
-	groupConversation
-	IsGroup bool `json:"is_group"`
+	GroupConversation
```

Comment:
I think you may have missed the point of adding GroupConversation. It includes the common members that are shared by both Channel and Group. You cannot be a member (`IsMember`) of a Channel. You cannot have a group that `IsGeneral`. `IsChannel` and `IsGroup` are mutually exclusive. Channels and Groups never come back in a unified list. Consequently, I'm not sure why you made the change?